go.mod

```diff
@@ -375,6 +374,7 @@ replace (
 	github.com/go-task/slim-sprig => github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
 	github.com/godbus/dbus/v5 => github.com/godbus/dbus/v5 v5.0.6
 	github.com/gofrs/uuid => github.com/gofrs/uuid v4.0.0+incompatible
+	github.com/gogo/googleapis => github.com/gogo/googleapis v1.4.1
```

Comment:
is this new? where did this come from?

Reply:
See https://github.com/google/cadvisor/pull/3145#discussion_r934795153

It is needed as part of pulling in the new docker, I believe this was the PR where it was added - https://github.com/moby/moby https://github.com/moby/moby/pull/43239/

Reply:
indeed!